examples/guardrails-evaluation/README.md

```diff
@@ -0,0 +1,163 @@
+# Guardrails Evaluation Preset
+
+This preset provides a comprehensive testing suite for evaluating guardrails effectiveness against the most critical security threats and harmful content that guardrails are designed to prevent.
```

Comment:
consider explaining why here. Read through https://huggingface.co/ibm-granite/granite-guardian-3.2-5b#evaluations which talks about datasets IBM used in a guardrails eval.